libs/zzz/db/src/Database/DataManagers/DiscDataManager.ts

```
@@ -344,23 +352,18 @@ function parseSubstats(
 ): ISubstat[] {
   if (!Array.isArray(obj)) return []
   const substats = (obj as ISubstat[])
-    .map(({ key = '', value = 0 }) => {
+    .map(({ key, upgrades = 0 }) => {
       if (!key) return null
       if (
         !allDiscSubStatKeys.includes(key as DiscSubStatKey) ||
-        typeof value !== 'number' ||
-        !isFinite(value)
+        typeof upgrades !== 'number' ||
+        !isFinite(upgrades)
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)